dom0/sd-clean-all.sls

```diff
@@ -99,12 +99,6 @@ sd-cleanup-etc-changes:
     - repl: ''
     - backup: no
 
-{% if d.environment == "prod" or d.environment == "staging" %}
```

Comment:
Do we even need a conditional on this if we keep it? Is there a downside to the restart on dev environments?